styles/custom.scss

```diff
@@ -71,6 +67,37 @@ $hr-opacity: 1;
   border: none !important;
 }
 
+.nav-shadow {
+  box-shadow: 0px 3px 6px $nav-shadow;
+  z-index: 3;
+}
+
+.nav-tile {
+  cursor: pointer;
+  border: 0.5px solid rgba(213, 213, 213, 0.4) !important;
+  box-shadow: 0px 3px 6px $nav-shadow;
+}
+
+.nav-tile:hover {
+  box-shadow: 0px 7px 10px $nav-shadow;
+}
+
+.card-img-top {
+  object-fit: cover;
+}
+
+.card-text {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3;
```

Comment:
ellipses truncation after 3 lines 🤝 side note - is using webkit for this okay? i see we use it elsewhere in VZE and moped

Reply:
@roseeichelmann wrote:
> is using webkit for this okay?

I say 100% yes. Chrome + safari are webkit, so that's a real good chunk of browsers right there. I'd say, if it works for you on Firefox on ubuntu, then it's going to almost always going to be a supported feature.

👍

Reply:
It does indeed work on my ubuntu + firefox setup, so I'm confident it should work for most! @frankhereford